### PR TITLE
Make extra Host callbacks triggering more readable

### PR DESCRIPTION
**Pull request by pitr-ch**
- just nice to have minor improvement
- [ ] not tested locally waiting for CI

### CODE REVIEW

File: app/models/host/managed.rb [hunk 1]
```diff
@@ -27,19 +27,20 @@ class Host::Managed < Host::Base
   define_model_callbacks :provision, :only => :before
 
   # Custom hooks will be executed after_commit
-  after_commit :build_hooks
+  after_commit :triggert_build_hooks
```
Comment (domcleal) [inline app/models/host/managed.rb]: Did you mean "trigger"?

**Comment by domcleal** [comment]
Whether this is more readable must be in the eye of the beholder, as I find it harder to read!

File: app/models/host/managed.rb (hunk 1 above)
Comment (pitr-ch) [inline app/models/host/managed.rb]: yeah sorry.

**Comment by pitr-ch** [comment]
Yeah benefits of this PR are uncertain :). I was digging into the old version and ti was hard for me to decode it. Previous implementation requires knowledge of the orchestration code: what is method `#old` returning, it is there only for `not new_record?` models. The implementation in the PR is done using only ActiveModel API and it's declaring more clearly what change needs to happen to trigger the callback.

**Comment by dLobatog** [comment]
@pitr-ch @domcleal Any updates on this? I would :-1: this too, sorry, current code reads fine, except perhaps for the first complex conditional

**Comment by pitr-ch** [comment]
Yeah exactly the first condition is the problem. When you do not know when the callbacks are called it's really hard to decipher from the source code. Rebased to latest master. I am not sure why not to merge, it's just formatting change for the better. No functionality change.

**Comment by ohadlevy** [comment]
closing due to lack of activity, reopen if any update on the pr. thanks

**Pull request closed without merging**